.github/workflows/lint.yaml

```diff
@@ -26,20 +26,63 @@ jobs:
     name: Shellcheck
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.0.0
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.0
+
       - name: Run Shellcheck
         uses: Seagate/action-shellcheck@203a3fd018dfe73f8ae7e3aa8da2c149a5f41c33
 
   solution_yaml:
     name: Solution YAML validation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.0.0
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.0
+
       - name: Validate solution.example.yaml
         run: ./solution_validation_scripts/solution-validation.sh solution.example.yaml
         shell: bash
         working-directory: ./k8_cortx_cloud
+
       - name: Validate solution_stub.yaml
         run: ./solution_validation_scripts/solution-validation.sh solution_stub.yaml
         shell: bash
         working-directory: ./k8_cortx_cloud
+
+  # Reference: https://github.com/helm/chart-testing-action
+  chart-lint-test:
+    name: Lint and Test Helm Charts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.0
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
```

Comment:
Need to fork this into Seagate?

Reply:
Yep. Even though this project looks well maintained by Microsoft, we'll want to make sure we control what code is running through each of our pipeline runs.

Reply:
I'll submit a request to have this mirrored, but leave it for now until that happens.